src/solver/conv_hip_implicit_gemm_3d_grouped_fwd_xdlops.cpp

```diff
@@ -314,7 +320,7 @@ bool ConvHipImplicitGemm3DGroupFwdXdlops::IsApplicable(
     if(!problem.IsLayoutNHWC())
         return false;
     const std::string& arch = ctx.GetStream().GetDeviceName();
-    if(!(arch == "gfx908" || arch == "gfx90a"))
+    if(miopen::StartsWith(arch, "gfx11") || miopen::StartsWith(arch, "gfx10"))
```

Comment:
I would suggest we convert this to a white list, so, list the arches that CK supports.

Reply:
something similar to https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/src/include/miopen/solver/ck_utility_common.hpp#L49

Reply:
Good suggestion and should be used in all CK-based solvers.

Reply:
> I would suggest we convert this to a white list, so, list the arches that CK supports.

Great idea. I have applied the change to all ck based solvers and currently doing the local test before push the changes.